src/Core/Feature.js

```diff
@@ -273,6 +273,16 @@ export class FeatureCollection {
         this.features.push(feature);
         this.updateExtent(feature.extent);
     }
+    getFeature(type, callback) {
+        const feature = this.features.find(callback);
+        if (feature && this.optionsFeature.mergeFeatures) {
```

Comment:
It should return the `feature` if it is found, if `mergeFeatures` is not defined, so you get something like this:
```js
if (feature && (this.optionsFeature.mergeFeatures == undefined || this.optionsFeature.mergeFeatures)) {
```
or else you should rename the function as it does not really get the Feature but only the Feature if the Features are merged.